examples/shared_samples/fc_network.rb

```diff
@@ -32,6 +40,15 @@
 # api_version = 500 & variant = Synergy to OneviewSDK::API500::C7000::FCNetwork
 # api_version = 600 & variant = C7000 to OneviewSDK::API600::C7000::FCNetwork
 # api_version = 600 & variant = Synergy to OneviewSDK::API600::C7000::FCNetwork
+# api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::FCNetwork
+# api_version = 800 & variant = Synergy to OneviewSDK::API800::C7000::FCNetwork
```

Comment:
It should be OneviewSDK::API800::Synergy::FCNetwork

Reply:
Fixed all together